src/main/java/Duke.java

```diff
@@ -1,10 +1,39 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Creates a new instance of Duke.
+     */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage("data/tasks.txt");
```

Comment:
"Avoid magic numbers" https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers

Might want to use a named constant for this?